cves/CVE-2019-12781.yml

```diff
@@ -116,10 +130,12 @@ discovered:
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    It seems the initial vulnerability was reported by someone named Gavin Wahl.
+    He is also credited the initial suggestion for the patch.
```

Comment:
Consider adding a link to where Gavin Wahl is being credited (bug report, security release, etc.)